### ISSUE TITLE
Incorrect type. Expected "string".

### ISSUE THREAD

**Issue by definitio** [keesschollaart81/vscode-home-assistant#54]
```
automation:
  - alias: 'test'
    trigger:
      - platform: homeassistant
        event: start
    action:
      - wait_template: "{{ is_state('binary_sensor.entrance', 'on') }}"
        timeout:
          minutes: 5
```
`minutes: 5` is highlighted, although it is a correct syntax.

**Comment by keesschollaart81** [comment]
Fixed with v1.0.1

**Comment by definitio** [comment]
Thanks
